architecture-building-systems/CityEnergyAnalyst
# 3137 VCC model update

This change brings the VCC model back to its original state. Where the VCC's COP is assumed to correspond to a certain percentage of the Carnot COP. This model clearly overestimates the efficiency of VCCs, but in contrast to the formerly established model is at least mathematically sound. I will attempt to introduce a more realistic model in the future.

To test simply:
1. Run the district cooling unit test in debug  mode, with a breakpoint set at line 183.
2. Once the debugger reaches line 183 compare the test values to the expected values. 

The values should differ, since I have not yet updated the expected values (they are still the same as in the current master branch). You should be able to observe:
A. that the electricity demand of the VCCs is significantly lower than before. This can be explained through the overestimation of the COP that the new model calculates.
B. that there is a slight change in most cooling dispatch amounts (except Tirgen direct load). This is due to a higher cooling load being covered by the WS VCCs in cases, where the water body cooling potential is restricted and the resulting shift in the activation chain.

Once you confirm these observations and agree to continuing with the merge, I will update the expected results of the unit test.
